VectorDrawable2Svg.py

```diff
@@ -83,6 +83,9 @@ def convert_paths(vd_container, svg_container, svg_xml):
             if vd_path.hasAttribute('android:strokeColor'):
                 svg_path.attributes['stroke'] = get_color(
                     vd_path.attributes['android:strokeColor'].value)
+            if vd_path.hasAttribute('android:fillType'):
```

Comment:
Mind moving this up to just under the section for 'android:fillColor', around line 70?